File: TrabalhoParteFinal/src/disciplina/Disciplina.java
package src.disciplina;

import java.io.Serializable;

import src.aluno.AlunoMap;

public class Disciplina implements Serializable {
    private String nome;
    private AlunoMap alunos = new AlunoMap();

    public AlunoMap getAlunoMap() {
        return alunos;
    }

    public void setDisciplina(String nome) {
        this.nome = nome;
    }

    public String getNome() {
        return nome;
    }

}

File: TrabalhoC++/Disciplina.cpp
#include "Disciplina.hpp"

AlunoMap &Disciplina::getAlunoMap()
{
    return alunos;
}

const AlunoMap &Disciplina::getAlunoMap() const
{
    return alunos;
}

void Disciplina::setDisciplina(const std::string &nome)
{
    this->nome = nome;
}

std::string Disciplina::getNome() const
{
    return nome;
}
